### PR TITLE
Feature/enable hmr config envs

### PR DESCRIPTION
**Pull request by TheLarkInn**
HMR Baby

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
 	"presets": [
-		"react",
 		"es2015",
 		"stage-1",
-		"stage-2"
+		"stage-2",
+		"react"
```
Comment (blainekasten) [inline .babelrc:6]: you should make this alphabetical order

File: .editorconfig [hunk 2]
```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-indent_style = tab
+indent_style = space
```
Comment (blainekasten) [inline .editorconfig:4]: 👏

File: src/App/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { body, header } from './style.css';
 import React from 'react';
 import Navigation from './Navigation';
 
-const PULSE_LABZ = 'PULSE-labz';
+const PULSE_LABZ = 'PULSE-labzs';
```
Comment (blainekasten) [inline src/App/index.js:6]: lol

File: src/index.js [hunk 4]
```diff
@@ -7,14 +8,21 @@ import Home from './Home';
...
 		<Route path="/" component={App}>
 			<IndexRoute component={Home} />
 			<Route path="about" component={About} />
 			<Route path="contact" component={Contact} />
 			<Route path="oss" component={OSS} />
 		</Route>
-	</Router>,
-	document.getElementById('app')
+	</Router>
+  </AppContainer>
```
Comment (blainekasten) [inline src/index.js:21]: indentation is off here.

**Review by blainekasten — COMMENTED** [review]
Overall, PR looks good. Currently failing in travis

**Comment by TheLarkInn** [comment]
@pulse-opensource/owners Checks are now passing.

**Comment by TheLarkInn** [comment]
I'll come back in and clean up the babel and other config stuff. There are two babel configs one is for jest/webpack (to consume `.babel.js` files) and the one in sidie the loader config is for webpack.

File: .editorconfig (hunk 2 above)
Comment (colevoss) [inline .editorconfig:4]: 👏

File: package.json [hunk 5]
```diff
@@ -23,6 +23,16 @@
   },
   "homepage": "https://github.com/pulse-opensource/pulselabs-homepage#readme",
   "dependencies": {
+    "babel-preset-latest": "^6.16.0",
+    "preact": "^6.4.0",
```
Comment (colevoss) [inline package.json:27]: :|

File: src/App/index.js (hunk 3 above)
Comment (colevoss) [inline src/App/index.js:6]: yep

**Review by colevoss — APPROVED** [review]

File: package.json (hunk 5 above)
Comment (TheLarkInn) [reply package.json:27]: Yarn I'll completely wipe and reset these

**Pull request merged**